app/components/message-list.js

```diff
@@ -24,6 +24,10 @@ export default class MessageList extends Component {
     return this.state.dateSource.cloneWithRows(messages);
   }
 
+  _onRefresh() {
+    this.props.refreshView();
+  }
+
```

Comment:
Kunne kanskje bare sendt inn `this.props.refreshView` direkte til `RefreshControl` istedenfor å bruke denne funksjonen?